### PR TITLE
Matching game cards component #20

### PR DESCRIPTION
**Pull request by ssinikoglu**
Please review. This PR consists of matching game card and matching game cards into words and images categories. I will also make individual PR for matching game container.

Note: Matching Game needs development in these areas;

Drop words to the target,
Check words and image are correct match,
Cards layout styling to responsive

### CODE REVIEW

File: src/components/shuffle.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const shuffle = (arr) => {
+  const copyOfArr = [...arr];
+  for (let i = copyOfArr.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [copyOfArr[i], copyOfArr[j]] = [copyOfArr[j], copyOfArr[i]];
+  }
+  return copyOfArr;
+};
+
+export default shuffle;
```
[the remark this answers is not in the document]
Comment (louisrli) [reply src/components/shuffle.jsx:10]: tarkan already wrote this code, you two should reuse the same file (unless this is the same one and it just hasn't been committed)

File: src/components/MatchingGameImageCards/index.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ...t("games.matchingGame.matchingGameCards", {
+      returnObjects: true,
+    }),
+  ];
+  const shuffledMatchingGameCardsData = shuffle(matchingGameCardsData);
+  return (
+    <>
+      {shuffledMatchingGameCardsData.map((card) => (
+        <MatchingGameCard
+          id="matchingGame__imagesBoard__card"
```
Comment (majdajroudi) [inline src/components/MatchingGameImageCards/index.jsx:18]: id should be unique

File: src/components/shuffle.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (majdajroudi) [reply src/components/shuffle.jsx:10]: I realized that shuffle.jsx is stored directly inside components folder, I think it should stored in one of the following ways: [src/components/Shuffle/index.jsx](url) or [src/components/HelperFunctions/shuffle.jsx](url)

**Review by majdajroudi — COMMENTED** [review]
sorry for the late review. Generally good but needs some variable names changes for the readability of the code and explaining some points like `dataTransfer.setData` would be helpful for my code and probably other team members

**Comment by ssinikoglu** [comment]
Thanks for reviewing my code @louisrli @majdajroudi @tmansuroglu ! Sorry for not updating the PR according to your comments,  because of I was working on Contact page. I will check them today.

**Pull request closed without merging**